src/main/java/controllers/CommandLineInterface.java

```diff
@@ -201,6 +292,9 @@ public static void displayUserSchedules(List<Schedule> userSchedules) {
             System.out.println("Press '>' to go to the next schedule.");
             System.out.println("Press '<' to go to the previous schedule.");
             System.out.println("Press 'S' to save this schedule as an .ics file");
+            if (this.generationMode == 1) {
+                System.out.println("Press 'X' to build courses around this schedule");
```

Comment:
X seems kinda random? Also I don't know what this means by reading this output. This basically lets u use this schedule as a base and then add courses one by one, choosing which timeslot to add as you go right? Maybe be a bit more verbose.